app/controllers/SubmittedController.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import models.errors.{BadRequestError, Errors, InternalServiceError, NotFoundError}
+import play.api.libs.json.{JsObject, JsValue, Json}
+import play.api.mvc.{Action, AnyContent, ControllerComponents}
+import services.SubmittedService
+import uk.gov.hmrc.play.bootstrap.controller.BackendController
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@Singleton
+class SubmittedController @Inject()(submittedService: SubmittedService, cc: ControllerComponents) extends BackendController(cc) {
```

Comment:
Looking at the existing code; ScratchController PublishedController, I was expecting this to be named ApprovalController, so this would handle the submission and update of processes during the approval stage.

Reply:
I'm happy to change name, I called it Submitted as that lined up with the mongo collection names of scratch, published and submitted.
As we have changed the manage front end to Approval, I can rework this repo as well